docs/api/QUICKSTART.md

```diff
@@ -49,12 +49,12 @@ int main(void)
 {
     while (true) {
         led = !led;
-        Thread::wait(500);
```

Comment:
@AlessandroA @AnotherButler before accepting this pull request we must verify whether the proposed change to floating-point is resolved in the pre-compiler and not during runtime. uVisor currently doesn't support floating point numbers during runtime.